source/byte_buf.c

```diff
@@ -414,9 +414,7 @@ int aws_byte_buf_append_with_lookup(
         return AWS_OP_ERR;
     }
 
-    AWS_POSTCONDITION(aws_byte_buf_is_valid(to));
-    AWS_POSTCONDITION(aws_byte_cursor_is_valid(from));
-    return AWS_OP_SUCCESS;
+    return AWS_RETURN_SUCCESS_WITH_POSTCONDITION(aws_byte_cursor_is_valid(from) && aws_byte_buf_is_valid(to));
 }
```

Comment:
Is this macro really easier to read that the original code?  It is done, maybe not worth changing, but I would reconsider this style in future modules.